src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -1341,7 +1341,7 @@ internal static bool CheckManagedAddr(CSharpCompilation compilation, TypeSymbol
                     return false;
             }
         }
-#nullable restore
```

Comment:
What was wrong with leaving `#nullable restore` to terminate a nullable-enabled section?

Reply:
This will restore back to the project setting hence that is now effectively `#nullable enable`.